src/helper_bitcoin.py

```diff
@@ -1,10 +1,17 @@
+"""
+Calculates bitcoin and testnet address from pubkey
+"""
+
 import hashlib
+
 from pyelliptic import arithmetic
 
-# This function expects that pubkey begin with \x04
+
 def calculateBitcoinAddressFromPubkey(pubkey):
+    """This function expects that pubkey begin with \x04"""
```

Comment:
bad docstring

Reply:
Get the docstring from 
https://pybitmessage-test.readthedocs.io/en/latest/autodoc/pybitmessage.helper_bitcoin.html

Reply:
Impossible. Because that doc was generated from this source (my testing branch which has this fix#9 merged).

Reply:
This line is a dev comment and not describes the function.

Reply:
Okay, got it. 
Is new docstring Okay?

Reply:
No. It should be like """Calculate bitcoin address from given pubkey (65 bytes long hex string)"""

Reply:
Also it differs from the next function by one line

Reply:
Thank You :)